feat: Update seat usage calculation for Active Pro PLUTO-882 (#2044)
codacy/docs: docs/organizations/managing-people.md

To list and manage the people in your organization, open your organization **Settings**, page **People**. This page also shows their last activity on Codacy.

!!! note
    In some Enterprise plans, Codacy automatically manages seat usage for your organization based on the contribution to your private repositories:

>     -   Adds new seats for people who commit to your private repositories and analyzes their commits.

Reviewer: The sentence structure (the commit/analyzes combination) is a bit unclear. Could this make it clearer?

```suggestion
    -   Automatically adds a seat when analyzing the first contribution to your private repositories by a new person
```

    -   Deactivates the seats of people who perform no commit or have no commits analyzed for 90 consecutive days.

>     -   Only people committing to your private repositories or having their commits analyzed within the last 90 days occupy a seat in your organization.

Reviewer: Could the two points above be summarized by this one? It seems pretty clear already. Perhaps with a slight tweak:

```suggestion
    -   A person with at least one analyzed commit to your private repositories over the past 90 days occupies one seat in your organization.
```

Could we simplify it like this?
```
In some Enterprise plans, Codacy automatically manages seat usage by assigning a seat to any person with at least one analyzed commit to your private repositories over the past 90 days.
```

Reviewer: Perhaps with a further tweak to clarify that seats are also freed up?

```
In some Enterprise plans, Codacy automatically assigns a seat to any person with at least one analyzed commit to your private repositories over the past 90 days and frees up a seat after 90 days of inactivity.
```

Reviewer 2: I agree with merging the sentences. Since "having their commits analyzed" is just a consequence of a person that commited in the last 90 days. It's the same action

Author: > I agree with merging the sentences. Since "having their commits analyzed" is just a consequence of a person that committed in the last 90 days. It's the same action

@inesgomas "Having their commits analyzed" might not be a consequence of a person committing, because someone else can at a later date re-analyze a commit which I am the author of.

Anyway, we can merge the sentences in that way. 👍

Reviewer 2: can a person re-analyze a commit for an inactive seat? (a person that made that commit 90 days ago? I was expecting to not be possible 🤔 

Also it's not the fact that the commit was analyzed or not that affects your activity, it is the date of your commit

Reviewer: @claudiacarpinteiro Good point. Worth clarifying with an extra sentence?
```
Note that as a consequence, manually reanalyzing a commit older than 90 days may cause Codacy to reassign a freed-up seat.
```

Author: That's exactly the point! From what I understood, it's the date of the analysis, but I might be wrong... 🤔 
Calling @RaquelHipolito and @franciscoovazevedo for the rescue. 😉

Author: > @claudiacarpinteiro Good point. Worth clarifying with an extra sentence?
> 
> ```
> Note that as a consequence, manually reanalyzing a commit older than 90 days may cause Codacy to reassign a freed-up seat.
> ```
> 
> (provided this is true...)

Thank you @nicklem! Let's wait for the clarification of which date should matter.

Meanwhile, I applied your feedback to simplify the admonition content (with some tweaks):
`In some Enterprise plans, Codacy automatically assigns a seat to any person with at least one analyzed commit to your private repositories over the past 90 days and frees up the seat after 90 days of code analysis inactivity.`

Author: It seems that the activity period is affected only by the commit date. I'll proceed with the current update for now, and we can later re-iterate if needed.
